### ISSUE TITLE
Plaid SDK will not compile on app targeting SDK 31

### ISSUE THREAD

**Issue by zsperske** [plaid/plaid-link-android#205]
Android 12 requires any activity with an intent filter define an explicit value for `android:exported`. When trying to compile using the latest plaid SDK, the following activity is merged into the manifest.

```
<activity android:name="com.plaid.link.internal.LinkRedirectActivity" >
            <intent-filter>
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />

                <data
                    android:host="redirect"
                    android:scheme="plaid" />
            </intent-filter>
        </activity>
```

The following error is thrown:

```
android/app/src/main/AndroidManifest.xml Error:
        android:exported needs to be explicitly specified for <activity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
```

**Comment by JeroenMols** [comment]
What version of the SDK are you using?
Can you try upgrading to our latest version `3.6.2`? (this should be fixed since `3.6.0`)

**Comment by bridgingamit18** [comment]
Yes in this latest version it's fixed. I m using 3.3.0. 
And also I have updated the latest version.

Thanks
